package/skelcd-control-SLES.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Dec  4 18:29:36 UTC 2019 - Ladislav Slezák <lslezak@suse.cz>
+
+- Run the registration also on the Full medium (jsc#SLE-7214)
```

Comment:
How it will looks like from user perspective? does he get same warnings that system is not registered and so on? I think that at least some better wording and maybe new text is needed, like that updates should be done by that new quaterly update media? @jsrain ?

Reply:
If you register on the Full medium it should behave the same way as the Online medium. (Just in that case skip offering the modules/extensions from DVD.)